manager/controllers/default/security/login.class.php

```diff
@@ -90,7 +90,7 @@ public function process(array $scriptProperties = []) {
         }
         $this->setPlaceholder('background', $background);
 
-        $logo = $this->modx->getOption('login_logo', null, $managerUrl . 'templates/' . $managerTheme . '/images/modx-logo-color.svg', true);
+        $logo = $this->modx->getOption('login_logo', null, $managerUrl . 'templates/' . $managerTheme . '/images/modx-logo-color.png', true);
```

Comment:
This is actually the wrong place to change this, as this is for the web UI not the email html. You need to alter the email template at `manager/templates/default/email/default.tpl`. But, it'd be preferable to provide the png as a fallback to the svg, maybe trying something like:
```html
<img 
     src="{$_config.url_scheme}{$_config.http_host}{$_config.manager_url}templates/default/images/modx-logo-color.png" 
     alt="{$_config.site_name}"
     srcset="{$_config.url_scheme}{$_config.http_host}{$_config.manager_url}templates/default/images/modx-logo-color.svg" 
>
```